---
layout: post
title: DSN_normalization_7_29_21
date: '2021-07-29'
categories: Goals
tags: 
---

7-29-2021
### DSN Normalization

DSN normalization is critical ensuring an even distribution of coverage across probes.  There are a genes that are highly expressed in all cells and DSN normalization helps to remove these high abundance probes and transcripts.  

#### DSN needs to be properly dilued and should be tested for activity levels before proceeding

#### The protocol below was taken from Illumina's recommendations [LINK](https://github.com/MarineEvoEcoLab/EecSeq/blob/master/DSN_Normalization_SamplePrep_Guide_15014673_B.pdf)
#### Reagents

| Reagent| Supplier|
|----------|--------------|
|1 M HEPES buffer solution|Invitrogen, part # 15630‚Äê080 |
|5 M NaCl solution|Ambion, part # AM9760G|
|KAPA HiFi HotStart PCR kit with dNTPs|Kapa, part #KK2502|
|Strip tubes|General lab supplier|
|DSN Kit|Evrogen, part # EA001 Sigma Aldrich, part # E7023|
|Ethanol 200 proof (absolute) for molecular biology (500 ml)|AB, part # 4333764F|
|10X KAPA Library Amplification Primer Mix (Universal primer mix)|Included in KAPA stranded mRNA kit|
|SPRI beads|KAPA Pure Beads, part #KK8000|
|Nuclease-free water|General lab supplier|

#### Equipment
* Thermocycler (2!)
* Magentic stand compatible with strip tubes

#### Procedure

1. First pool individual RNA libraries in equal quantities to create a single pool, we have experimented with 500 ng total or 200ng of each library, the final volume of this should not exceed 13.5 Œºl. If many probes are desired, multiple probe synthesis reactions can be done, and pooled at the end before hybridization

2. Create a 4X hybridization solution (this amount is excess of what is needed):

|Component|Volume|
|---------|------|
|1 M HEPES buffer solution| 200 ùúál|
|5 M NaCl solution| 400 ùúál|
|Nuclease‚Äêfree water| 400 ùúáL|
|**Total Volume**|**1000 ùúá**L|

3.  Use two thermocyclers and set one to hold at 68¬∞C
4. Prepare the following reaction mix in a separate, sterile, nuclease‚Äêfree 200 Œºl PCR tube on ice for each pool of cDNA to be normalized. If the pooled cDNA is less than 13.5 Œºl, increase the volume to 13.5 Œºl with nuclease free water:

|Component|Volume|
|---------|------|
|Sample library | 13.5 ùúál|
|4X Hybridization buffer| 4.5 ùúál|
|**Total Volume Per Sample**|**18 ùúáL**|

5. Gently pipette the entire volume up and down 10 times, then centrifuge briefly to mix
6. Transfer the entire volume of reaction mix directly to the bottom of a new, sterile, nuclease‚Äêfree 200 Œºl PCR tube, using a pipette. Do not let the sample contact the side of the tube during the process
7. Incubate the reaction mix tube on the thermal cycler using the following PCR cycling conditions:

|Step|Temp|Duration|
|----|----|--------|
|Initial denaturation|98 ¬∞C|2 min|
|Treatment|68 ¬∞C|5 hours|

* **Caution**- Following incubation, keep the thermal cycler lid closed and the temperature held at 68¬∞C. Do not remove the reaction mix tube from thermal cycler prior to and during DSN treatment.
8. Dilute the 10X DSN Master buffer supplied in the DSN kit to 2X with nuclease‚Äê free water by combining 16 Œºl nuclease free water and 4 Œºl 10X DSN master buffer for a total volume of 20 Œºl
9. Pre‚Äêheat the 2X DSN buffer on the pre‚Äêheated heat block (or another thermocycler) at 68¬∞C
  * **Note:** Do not remove the 2X DSN buffer from the heat block during DSN treatment, pipette from the block or thermocycler
10. Quickly add 20 Œºl of pre‚Äêheated 2X DSN buffer to the first reaction mix tube
13. With the reaction mix tube remaining within the thermal cycler, gently pipette the entire volume up and down 10 times to mix thoroughly using a pipette set to 40 Œºl.
  * **Note**:Pipette the solution directly to the bottom of the PCR tube and do not let the sample contact the side of the tube during the process.
  * **Note**: It is important to keep the thermal cycler closed, except for the time necessary to add the 2X DSN buffer and mix. When preparing more than one reaction mix tube, keep the thermal cycler lid closed when extracting the 2X DSN buffer from its tube, then open the thermal cycler lid only for the time necessary to add and mix the 2X DSN buffer.
14. Repeat steps 2 and 3 for each reaction mix tube
15. Incubate the reaction mix tubes on the thermal cycler at 68¬∞C for 10 minutes.
16. Quickly add 2 Œºl of DSN enzyme to the first reaction mix tube using a 2 Œºl pipette
17. With the reaction mix tube remaining within the thermal cycler, gently pipette the entire volume up and down 10 times to mix thoroughly using a pipette set to 40 Œºl
  * **Note**:Pipette the solution directly to the bottom of the PCR tube and do not let the sample get stuck on the side of the tube during the process
18. Repeat steps 6 and 7 for each reaction mix tube
19. Incubate the reaction mix tubes on the thermal cycler at 68¬∞C for 25 minutes
20. Add 40 Œºl of 2X DSN stop solution (should be thawed but on ice) to each reaction mix tube. Gently pipette the entire volume up and down to mix thoroughly, then place the tubes on ice

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‚Äê15¬∞ to ‚Äê25¬∞C.

---
### 1.6X Cleanup

1. Bring KAPA Pure Beads to room temperature by taking out of the 4 degree ~30 minutes before use and swirl to mix
2. Perform a 1.6X bead cleanup by adding 128 Œºl of KAPA Pure Beads to each DSN treated pool
3. Thoroughly resuspend the beads by pipetting up and down multiple times
4. Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if the samples can be placed on an orbital shaker at 200rpm this is ideal
5. Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear
6. Carefully remove and discard ~200 Œºl of supernatant
7. Keeping the plate/tube on the magnet, add 200 Œºl of 80% ethanol
8. Incubate the plate/tube at room temperature for ‚â•30 sec.
9. Carefully remove and discard the ethanol
10. Keeping the plate/tube on the magnet, add 200 Œºl of 80% ethanol
11. Incubate the plate/tube at room temperature for ‚â•30 sec
12. Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads
13. Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss**
14. Remove the plate/tube from the magnet
15. Thoroughly resuspend the beads in 25 Œºl of 10 mM Tris-HCl (pH 8.0)
16. Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads, if the samples can be placed on an orbital shaker at 200rpm this is ideal
17. Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear
18. Transfer 24 Œºl of the clear supernatant to a new plate/tube and proceed to next step

### PCR Enrichment of DSN Normalized Library

1. Create the reaction mix by combining these components on ice _note, if more than 1 DSN normalization took place, you can do multiple enrichments, or you could have combined them before in the cleanup step above and adjusted the cleanup volume_:

|Component|Volume|
|---------|------|
|DSN Treated Library | 30 Œºl|
|2X KAPA HiFi HotStart ReadyMix| 25 Œºl|
|10X KAPA Library Amplification Primer Mix| 5 Œºl|
|**Total Volume per sample**| **50 Œºl**|

2. Mix well by pipetting up and down several times
3. Amplify the library using the following thermal cycling protocol:

|Step|Temp|Duration|Cycles|
|----|----|--------|------|
|Initial denaturation|98 ¬∞C|45 sec|1|
|Denaturation|98 ¬∞C|15 sec|12|
|Annealing*|60 ¬∞C|30 sec|12|
|Extension|72 ¬∞C|30 sec|12|
|Final Extension|72 ¬∞C|5 min|1|
|Hold|10 ¬∞C | ‚àû|1|

### 1.6X Cleanup of DSN Treated and Amplified Pooled cDNA

1. Bring KAPA Pure Beads to room temperature by taking out of the 4 degree ~30 minutes before use and swirl to mix
2. Perform a 1.6X bead cleanup by adding 80 Œºl of KAPA Pure Beads to each DSN treated pool
3. Thoroughly resuspend the beads by pipetting up and down multiple times
5. Incubate the plate/tube at room temperature for 15 min to allow the DNA to bind to the beads, if the samples can be placed on an orbital shaker at 200rpm this is ideal
6. Place the plate/tube on a magnet to capture the beads Incubate until the liquid is clear
7. Carefully remove and discard 115 Œºl of supernatant
8. Keeping the plate/tube on the magnet, add 200 Œºl of 80% ethanol
9. Incubate the plate/tube at room temperature for ‚â•30 sec
10. Carefully remove and discard the ethanol
11. Keeping the plate/tube on the magnet, add 200 Œºl of 80% ethanol
12. Incubate the plate/tube at room temperature for ‚â•30 sec
13. Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads
14. Dry the beads at room temperature, until all of the ethanol has evaporated. **Caution: over-drying the beads may result in dramatic yield loss**
15. Remove the plate/tube from the magnet
16. Thoroughly resuspend the beads in 22 Œºl of 10 mM Tris-HCl (pH 8.0)
17. Incubate the plate/tube at room temperature for 5 min to allow the DNA to elute off the beads, if the samples can be placed on an orbital shaker at 200rpm this is ideal
18. Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear
19. Transfer 22 Œºl of the clear supernatant to a new plate/tube and proceed to next step

## Quant DSN Treated and Amplified Pooled cDNA
**Procedure (Standard HS DNA protocol)**
1. Set up the required number of 0.5-mL tubes for standards and samples. The Qubit¬Æ RNA HS Assay requires 2 standards
2. Label the tube lids
3. Prepare the Qubit¬Æ working solution by diluting the Qubit¬Æ DNA HS Reagent 1:200 in Qubit¬Æ DNA HS Buffer. Use a clean plastic tube each time you prepare Qubit¬Æ working solution. **Do not mix the working solution in a glass container**
4. Add 190 ŒºL of Qubit¬Æ working solution to each of the tubes used for standards
5. Add 10 ŒºL of each Qubit¬Æ standard to the appropriate tube, then mix by vortexing 2‚Äì3 seconds. Be careful not to create bubbles
6. Add Qubit¬Æ working solution to individual assay tubes so that the final volume in each tube after adding sample is 200 ŒºL (we use 199ŒºL solution to 1 ŒºL sample)
7. Add each sample to the assay tubes containing the correct volume of Qubit¬Æ working solution, then mix by vortexing 2‚Äì3 seconds. The final volume in each tube should be 200 ŒºL
8. Allow all tubes to incubate at room temperature for 2 minutes
9. On the Home screen of the Qubit¬Æ 3.0 Fluorometer, press DNA, then select DNA: High Sensitivity as the assay type. The ‚ÄúRead standards‚Äù screen is displayed. Press Read Standards to proceed
10. Insert the tube containing Standard #1 into the sample chamber, close the lid, then press Read standard. When the reading is complete (~3 seconds), remove Standard #1
11. Insert the tube containing Standard #2 into the sample chamber, close the lid, then press Read standard. When the reading is complete, remove Standard #2
12. Press Run samples
13. On the assay screen, select the sample volume and units
14. Insert a sample tube into the sample chamber, close the lid, then press Read tube. When the reading is complete (~3 seconds), remove the sample tube
15. Repeat step last step until all samples have been read

---

## If you want to sequence the probes you can split finished DSN treated cDNA Pools
* One tube for sequencing
* One tube for probe synthesis

#### Note that this step is optional as sequening the probes directly is not necessary for exome capture analysis.  Alternatively, a portion of the mRNA library can be saved before DSN normalization for sequencing. After this step the adapters are taken off the libraries, so they cannot be read on a sequencer anymore.

---

### Safe Stopping Point
This is a safe stopping point. If you are stopping, store your sample at ‚Äê15¬∞ to ‚Äê25¬∞C.

---